3_forecast/src/prep_inputs.R

```diff
@@ -19,52 +19,27 @@ prep_inputs <- function(
   ref_Date <- as.Date(ref_date)
   calib_ndays <- as.integer(365.25*5) # 5 years
   start_calib <- ref_Date - as.difftime(calib_ndays, units='days')
-  start_analysis <- ref_Date - as.difftime(182, units='days')
   forecast_range <- as.integer(ifelse(nwm_model=='med', 10, 30))
   end_forecast <- ref_Date + as.difftime(forecast_range-1, units='days')
 
-  # ultimately we'd like to use the analysis data for some gap between retro and
-  # forecast flow predictions, but for now we have (1) no analysis data
-  # available yet and (2) the retro ends 12/31/2017, so for later models there
-  # are NAs if we only use retro plus a single reference date. SO: for now we'll
-  # create a substitude "analysis" dataset made of lag-0 "forecasts" from
-  # previous reference days. we'll make this analysis dataset cover the period
-  # from 6 months before the reference date up until the reference date (defined
-  # in start_analysis above). But we don't have this "analysis" data between
-  # 2016-11-08 and 2017-05-08, we we'll need to use the retro data for that
-  # period for this "analysis" substitute
-  nwm_analysis <- bind_rows(
-    nwm_retro %>% filter(date >= as.Date('2016-11-08'), date < min(nwm_forecast$valid_date)),
-    nwm_forecast %>% filter(ref_date == valid_date) %>% select(site_no, date = valid_date, flow))
```

Comment:
the deletions in this file look wrong to me - i think i just added this and want to keep it

Reply:
yeah I had some merge conflicts that I think I screwed up but I'm pushing up your changes now